components/edit-collective/sections/SetupTwoFactorAuth.js

```diff
@@ -339,6 +480,10 @@ const addAccountHasTwoFactorAuthData = graphql(accountHasTwoFactorAuthQuery, {
   }),
 });
 
-const addGraphql = compose(addTwoFactorAuthToAccountMutation, addAccountHasTwoFactorAuthData);
+const addGraphql = compose(
+  addTwoFactorAuthToAccountMutation,
+  removeTwoFactorAuthFromAccountMutation,
```

Comment:
Usually, in our common practices, the `add` is not about adding or removing something to the account. It's about adding the mutation to the component. To follow practices in the project it should be:

`addAddTwoFactorAuthToAccountMutation`
`addRemoveTwoFactorAuthToAccountMutation`

If that sounds stupid, you can avoid it using an alternative syntax:

```
const addGraphql = compose(
  graphql(addTwoFactorAuthFromAccountMutation, {
    name: 'removeTwoFactorAuthTokenFromIndividual',
    options: { context: API_V2_CONTEXT },
  }),
  ...
);
```